### PR TITLE
GUI-2457, create alarm modal, form validation.

### PR DESCRIPTION
**Pull request by coyote240**
https://eucalyptus.atlassian.net/browse/GUI-2457

Begins to establish a good and simple pattern for displaying validation messages.  Please review.

### CODE REVIEW

**Comment by aliceh** [comment]
@coyote240 Test failed because I was changing templates, I'll trigger a retest

**Comment by aliceh** [comment]
Retest this, please

**Comment by aliceh** [comment]
Retest this please

**Comment by aliceh** [comment]
Retest this please

File: eucaconsole/templates/cloudwatch/create_alarm_modal_template.pt [hunk 1]
```diff
@@ -57,18 +61,24 @@
         </div>
         <div class="small-3 columns">
             <input ng-model="alarm.threshold" name="threshold"
-                   pattern="^[1-9]\d*$" placeholder="amount..." required="required" type="number"/>
+                   placeholder="amount..." required="required" type="number" min="1"/>
+            <small class="error" ng-show="createAlarmForm.threshold.$touched && createAlarmForm.threshold.$error.required" i18n:translate="">Threshold value is required.</small>
```
Comment (dkavanagh) [inline eucaconsole/templates/cloudwatch/create_alarm_modal_template.pt]: Something tells me this needs to be escaped &amp;, but try "make translations" to see.

**Comment by aliceh** [comment]
Retest this please

**Pull request merged**